### PR TITLE
cmd/mount: add `prefix-internal` option to prefix internal files by .jfs

### LINKED ISSUE #3120: Disable/change virtual file names
**What would you like to be added**:

Please add an option to either rename the built-in virtual files (`.accesslog`, `.config`, etc.), or to disable them altogether when mounting the file-system.

**Why is this needed**:

I'm attempting to use JuiceFS as a volume backend for my docker containers. However, some of them (such as MongoDB) attempt to create a `.config` folder at the root of the volume, which conflicts with the `.config` virtual file (ironically, this file was meant to be named `.jfsconfig` initially, avoiding this issue - but was renamed to simply `.config` before PR #652 was finalized).

### PR DESCRIPTION
**Pull request by SandyXSD**
close #3120

### CODE REVIEW

File: cmd/mount.go [hunk 1]
```diff
@@ -328,7 +328,10 @@ func prepareMp(newCfg *vfs.Config, mp string) (ignore bool) {
 		return
 	}
 
-	contents, err := os.ReadFile(path.Join(mp, ".config"))
+	contents, err := os.ReadFile(filepath.Join(mp, ".jfs.config"))
```
Comment (davies) [inline cmd/mount.go:331]: create a function for this

File: cmd/debug.go [hunk 2]
```diff
@@ -512,12 +515,20 @@ func collectSysInfo(ctx *cli.Context, currDir string) error {
 }
 
 func collectSpecialFile(ctx *cli.Context, amp string, currDir string, requireRootPrivileges bool, wg *sync.WaitGroup) error {
-	configName := ".config"
+	prefixed := true
+	configName := ".jfs.config"
+	if _, err := os.Stat(filepath.Join(amp, configName)); os.IsNotExist(err) {
```
Comment (davies) [inline cmd/debug.go:520]: utils.Exists()

File: cmd/mount.go [hunk 3]
```diff
@@ -252,12 +251,13 @@ func expandPathForEmbedded(addr string) string {
...
-		Chunk:      chunkConf,
-		BackupMeta: duration(c.String("backup-meta")),
-		Port:       &vfs.Port{DebugAgent: debugAgent, PyroscopeAddr: c.String("pyroscope")},
+		Meta:           metaConf,
+		Format:         *format,
+		Version:        version.Version(),
+		Chunk:          chunkConf,
+		BackupMeta:     duration(c.String("backup-meta")),
+		Port:           &vfs.Port{DebugAgent: debugAgent, PyroscopeAddr: c.String("pyroscope")},
+		PrefixInternal: c.Bool("prefix-internal"),
```
Comment (davies) [inline cmd/mount.go:260]: internal-prefix ?

Comment (SandyXSD) [reply cmd/mount.go:260]: here `prefix` is a verb

File: pkg/vfs/vfs.go [hunk 4]
```diff
@@ -956,6 +957,12 @@ func NewVFS(conf *Config, m meta.Meta, store chunk.ChunkStore, registerer promet
 	if conf.Meta.Subdir != "" { // don't show trash directory
 		internalNodes = internalNodes[:len(internalNodes)-1]
 	}
+	if conf.PrefixInternal {
+		for _, n := range internalNodes {
+			n.name = ".jfs" + n.name
+		}
+		meta.TrashName = ".jfs" + meta.TrashName
```
[the remark this answers is not in the document]
Comment (davies) [reply pkg/vfs/vfs.go:964]: no problem

**Pull request merged**